daemon/monitor.go

```diff
@@ -127,6 +127,7 @@ func (daemon *Daemon) ProcessEvent(id string, e libcontainerd.EventType, ei libc
 				"container": c.ID,
 				"exec-pid":  ei.Pid,
 			}).Warnf("Ignoring Exit Event, no such exec command found")
+			panic("Unknown Exit ProcessID")
```

Comment:
I guess this would get called for example on live-reload, when exec was created before restart.

Reply:
Not planning to leave it here, just for testing :)

Reply:
I removed it. There's a race between this handler and the code that starts the exec. The latter will remove the exec in the defer on error which is the case when the context is cancelled. I wonder if this case should be left out to the Event Handler (that way we'd get the proper exit code too).

Force live-restore, I don't think it's an issue, we don't save the exec configs on disk, so we don't reload them.